### PR TITLE
Delete Guardian_Subscriptions.co.uk.xml

### PR DESCRIPTION
**Pull request by pipboy96**
nxdomain

### CODE REVIEW

File: src/chrome/content/rules/Guardian_Subscriptions.co.uk.xml [hunk 1]
```diff
@@ -1,26 +0,0 @@
-
-<!--
-Disabled by https-everywhere-checker because:
-Fetch error: http://guardiansubscriptions.co.uk/ => https://guardiansubscriptions.co.uk/: (28, 'Connection timed out after 20001 milliseconds')
-Fetch error: http://www.guardiansubscriptions.co.uk/ => https://www.guardiansubscriptions.co.uk/: (28, 'Connection timed out after 20001 milliseconds')
-
-	For other Guardian coverage, see Guardian.xml.
```
Comment (Bisaloo) [inline src/chrome/content/rules/Guardian_Subscriptions.co.uk.xml]: Can you remove the reference to this ruleset in `Guardian.xml` please?

Comment (pipboy96) [reply src/chrome/content/rules/Guardian_Subscriptions.co.uk.xml]: `Guardian.xml` only has a weird self-reference (It itself has a line `For other Guardian coverage, see Guardian.xml.`), no references to this ruleset.

Comment (Bisaloo) [reply src/chrome/content/rules/Guardian_Subscriptions.co.uk.xml]: Hum, the references indeed got mixed up. This is the ruleset you should edit: https://github.com/EFForg/https-everywhere/blob/master/src/chrome/content/rules/The_Guardian.com.xml

Comment (pipboy96) [reply src/chrome/content/rules/Guardian_Subscriptions.co.uk.xml]: @Bisaloo Done.

**Comment by Bisaloo** [comment]
@pipboy96 please have a look at the failing tests

**Comment by pipboy96** [comment]
@Bisaloo Done.

**Comment by Bisaloo** [comment]
The issues don't look too bad. Could you fix them instead of whitelisting the ruleset please?

**Comment by pipboy96** [comment]
@Bisaloo Done.

**Comment by Bisaloo** [comment]
Thank you!

**Pull request merged**